dolphinscheduler-ui-next/src/views/datasource/list/use-form.ts

```diff
@@ -138,9 +139,15 @@ export function useForm(id?: number) {
   const resetFieldsValue = () => {
     state.detailForm = { ...initialValues }
   }
-  const setFieldsValue = (values: object) => {
-    state.detailForm = { ...state.detailForm, ...values }
+
+  const setFieldsValue = (values: IDataSource) => {
+    state.detailForm = {
+      ...state.detailForm,
+      ...values,
+      other: JSON.stringify(values.other)
```

Comment:
`values.other` may be null.  You should judge it.

Reply:
i will fix it